index.html

```diff
@@ -2,9 +2,135 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <title>Title</title>
+    <title>NOVI pagina namaken</title>
+    <link rel="stylesheet" href="styles.css">
+
 </head>
 <body>
+<header>
+    <nav>
+        <ul>
+            <li>Case Studies</li>
+            <li>Services</li>
+            <li>Contact Us</li>
+        </ul>
+    </nav>
+    <div class="intro">
+        <h1>
+            Your website is an investment in the success of your business.
+        </h1>
+        <h3>
+            We are here to help you succeed.
+        </h3>
+        <button>Let's work together</button>
+        <div class="image">
+            <img src="assets/squiggle_line_stripe.svg">
+        </div>
+</header>
+<div class="intro2">
+    <h2>
+        <div>
+            Rekupe is a digital agency in Los Angeles.
+        </div>
+        <br>
+        <div>
+            We use strategic UX, SEO optimization and beautiful visual designs to help
+            your business grow.
+        </div>
+        <br>
+        <div>
+            We see every project as a fresh opportunity, pushing ourselves to create new digital experiences.
+        </div>
+
+
+    </h2>
+    <p class="contactLink">Have a question? <a href="">Click here</a> to contact us!</p>
+</div>
+<main>
+    <div class="items">
+        <div class="itemtekst">
+            <article class="item">
+                <h2 class="itemtitle">Rolling Stone</h2>
+                <img src="assets/squiggle_line.svg" alt="">
+                <p>You know Rolling Stone. It’s the definitive voice in music, politics and culture. It’s a global icon.</p>
+                <span class="imgwrapper">
+                <img class="itemimg" src="assets/work-image-01.jpeg" alt="">
+        </div>
+
+            </span>
+        </article>
```

Comment:
Volgens mij gaat het hier niet goed en moet het zo zijn
```suggestion
        </span>

            </article>
        </div>
```